smoke.sh

```diff
@@ -68,6 +72,8 @@ function finish ()
     if [ $RET -ne 0 ]; then
         cat /build/dbench-logs || true
     fi
+    #Move statedumps to be archived
+    mv /var/run/gluster/*dump* /var/log/glusterfs/
```

Comment:
That doesn't work for mounts. It works in conjunction with 'gluster volume statedump' which is for bricks